python-regression/tests/features/steps/local_snapshots_steps.py

```diff
@@ -75,3 +77,99 @@ def read_ls_metadata(step, node):
     for hash_val in hashes:
         assert hash_val in node_metadata, "Provided hash: {} was not found in the snapshot metadata".format(hash_val)
     logger.info("Snapshot Metadata contains the provided hashes.")
+
+
+@step(r'the spent addresses are exported from "([^"]+)"')
+def export_spent_addresses(step, node):
+    """
+    Uses an ixi module to export the spent addresses from the current node to a text file.
+
+    :param node: The node that the spent addresses will be pulled from
+    """
+    command = {"command": "export-spent.generateSpentAddressesFile"}
+    request_return = api_utils.send_ixi_request(node, command)
+    assert 'ixi' in request_return, "Error: {}".format(request_return['error'])
+
+    num_exports = request_return['ixi']['amount']
+    checksum = request_return['ixi']['checksum']
+
+    # Change these assertion value with the appropriate ones if you have used a custom db and not the provided one
+    expected_num_exports = 74
+    expected_checksum = "919432abf85f939057f020d1a589f4d5513ff8890b0fb6b8e2242f90fa9158bc"
+
+    assert num_exports == expected_num_exports, \
+        "Expected to export {} addresses, instead exported {}".format(expected_num_exports, num_exports)
+    assert checksum == expected_checksum, \
+        "The checksum {} does not match the expected: {}".format(checksum, expected_checksum)
+
+    logger.info("Spent addresses file exported as: {}".format(request_return['ixi']['fileName']))
+
+    try:
+        podname = world.machine['nodes'][node]['podname']
+        config.load_kube_config()
+        v1 = client.CoreV1Api()
+        response = stream(v1.connect_get_namespaced_pod_exec,
+                      podname, 'default',
+                      command=['cat', '/iri/data/spentAddresses.txt'],
+                      stderr=True, stdin=True, stdout=True, tty=False,
+                      _preload_content=False)
+
+        while response.is_open():
+            response.update(timeout=1)
+            if response.peek_stdout():
+                f = open("/tmp/spentAddresses.txt", "w+")
+                f.write(response.read_stdout())
+
+    except Exception as err:
+        logger.info("There was an error reading the spent addresses file " + str(err))
+
+
+@step(r'the spent addresses are imported on "([^"]+)" from:')
+def merge_spent_addresses_file(step, node):
+    """
+    Reads the spentAddresses.txt file using the specified path and file name in the Gherkin options, and merges it into
+    the provided node. Then the return values for the IXI call are checked to make sure no failures occurred.
+
+    :param node: The node that the spent addresses will be merged into
+    """
+    args = step.hashes
+    for x in range(len(args)):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider using enumerate instead of iterating with range and len](https://app.codacy.com/manual/GalRogozinski/iri/pullRequest?prid=3955655)